frontend/src/app/modules/scripts/scripts-page/scripts-page.component.ts

```diff
@@ -61,14 +61,21 @@ export class ScriptsPageComponent extends BaseComponent implements OnInit {
     public openCreateModal(): void {
         const dialogRef = this.dialog.open(CreateScriptModalComponent, {
             width: '450px',
-            height: '400px',
+            height: '320px',
         });
 
         dialogRef.componentInstance.scriptCreated.subscribe((newScript: ScriptDto) => {
             this.loadScripts(newScript.id);
         });
     }
 
+    public executeScript() {
+        this.scriptService.executeScript({ content: this.form.value.scriptContent }).subscribe(
+            () => this.notification.info('Script successfully executed'),
```

Comment:
Can you make the result show on the UI?
Either a simple table or string field (can show as a modal) or at the bottom of page